src/app/containers/PageHandlers/withVariant/getVariantRedirectUrl/index.test.js

```diff
@@ -0,0 +1,306 @@
+import Cookie from 'js-cookie';
+import {
+  getVariant,
+  servicesWithVariants,
+} from '#lib/utilities/variantHandler';
+import { articlePath, frontPagePath } from '#app/routes/regex';
+import getVariantRedirectUrl from './index';
+
+const serviceNames = Object.keys(servicesWithVariants);
+
+describe('getVariantRedirectUrl', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
```

Comment:
no more mocks to clear so you can remove this line

Reply:
Fixed!